test/README.md

```diff
@@ -14,26 +15,83 @@ Use `go test`:
 go test -v ./pkg/...
 ```
 
-## Running end-to-end tests
+_By default `go test` will not run [the e2e tests](#running-end-to-end-tests), which need [`-tags=e2e`](#running-end-to-end-tests) to be enabled._
 
-### Dependencies
+## Presubmit tests
 
-There's couple of things you need to install before running e2e tests locally.
+[`presubmit-tests.sh`](./presubmit-tests.sh) is the entry point for the [end-to-end tests](/test/e2e).
+
+This script, and consequently, the e2e tests will be run before every code submission. You can run these tests manually with:
 
 ```shell
+test/presubmit-tests.sh
+```
+
+_Note that to run `presubmit-tests.sh` or `e2e-tests.sh` scripts, you'll need kubernetes `kubetest` installed:_
```

Comment:
I suggest linking to the "environment requirements" below instead of showing the `go get` command.

Reply:
OK.